docs/modules/ROOT/pages/references.adoc

```diff
@@ -1,98 +1,115 @@
-. [[ref:57]] OSLO spec, sec 3.3.4
-. [[ref:58]] OSLO spec, sec 3.3.1
-. [[ref:59]] OSLO spec, sec 3.3.5
-. [[ref:60]] OSLO spec, sec 3.3.3
-. [[ref:61]] OSLO spec, sec 3.3.2
-. [[ref:62]] OSLO spec, sec 3.2.5
-. [[ref:63]] OSLO spec, sec 3.1.11
-. [[ref:64]] Lemon
+. [[ref:1]] Codreanu, D. (2012) 10 rules for persistent uris, Joinup. Available at: https://joinup.ec.europa.eu/collection/semantic-interoperability-community-semic/document/10-rules-persistent-uris.
+. [[ref:2]] A. Miles and S. Bechhofer. SKOS simple knowledge organization system reference. W3C recommendation, W3C, Aug. 2009.
+http://www.w3.org/TR/2009/REC-skos-reference-20090818/.
+. [[ref:3]] E. Costetchi. eProcurement uml conceptual model to owl ontology transformation. Recommendation, Publications Office of the European Union, April
+2020
+. [[ref:4]] Gruber, T. (1997) What is an ontology? Stanford University. Available at: http://www-ksl.stanford.edu/kst/what-is-an-ontology.html.
+. [[ref:5]] Ontology (2023) Wikipedia. Wikimedia Foundation. Available at: https://en.wikipedia.org/wiki/Ontology (Accessed: January 2, 2023).
+. [[ref:6]] Barthelemy, F. et al. (2022) Core Person Vocabulary, Core person vocabulary. van Hooland, Seth - European Commission. Available at: https://semiceu.github.io/Core-Person-Vocabulary/releases/2.00/.
+. [[ref:7]] Barthelemy, F. et al. (2022) Core Public Service Vocabulary Application Profile (CPSV-AP). Available at: https://semiceu.github.io/CPSV-AP/releases/3.1.0/.
+. [[ref:8]] Dekkers , M. (2013) Asset Description Metadata Schema (ADMS), Asset description metadata schema (ADMS). W3C. Available at: https://www.w3.org/TR/vocab-adms/.
+. [[ref:9]] VOCABULARIES (2015). W3C. Available at: https://www.w3.org/standards/semanticweb/ontology#:~:text=On%20the%20Semantic%20Web%2C%20vocabularies,represent%20an%20area%20of%20concern (Accessed: January 2, 2023).
+. [[ref:10]] PwC EU Services (2017) DCAT Application Profile for data portals in Europe, Joinup. © 2015 European Union. Available at: https://joinup.ec.europa.eu/collection/semantic-interoperability-community-semic/solution/dcat-application-profile-data-portals-europe (Accessed: January 2, 2023).
+. [[ref:11]] Coyle, K. and Baker, T. (2009) Guidelines for Dublin Core™ Application Profiles. Available at: https://www.dublincore.org/specifications/dublin-core/profile-guidelines/.
+. [[ref:12]] C. Partridge, A. Mitchell, and S. de Cesare. Guidelines for developing ontological architectures in modelling and simulation. In Ontology, Epistemology, and
+Teleology for Modeling and Simulation, pages 27–57. Springer, 2013.
+. [[ref:13]] R. Soley et al. Model driven architecture. OMG white paper, 308(308):5, 2000.
+
+. [[ref:14]] J. Siegel. Object management group model driven architecture (mda) mda guide
+rev. 2.0. 2014. URL http://www.omg.org/cgi-bin/doc?ormsc/14-06-01
+. [[ref:15]] Semantic Interoperability Community. Available at: https://github.com/SEMICeu (Accessed: January 3, 2023).
```

Comment:
*Available at Link* is a very, very good idea. 
But it has to be formatted consistently with functional hyperlinks. (not sure if Antora will pick this one and make it functional )